<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroPage</class>
 <widget class="QWizardPage" name="IntroPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Intro Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>What should I generate:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="m_abstraction">
        <property name="text">
         <string>Abstraction</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_plugin">
        <property name="text">
         <string>Plugin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_node">
        <property name="text">
         <string>Node</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
